gnucash/import-export/import-backend.c

```diff
@@ -312,24 +311,20 @@ GdkPixbuf* gen_probability_pixbuf(gint score_original, GNCImportSettings *settin
     const gint num_colors = 5;
     gchar * size_str;
     gchar * none_color_str = g_strdup_printf("  c None");
-    gchar * green_color_str = g_strdup_printf("g c green");
-    gchar * yellow_color_str = g_strdup_printf("y c yellow");
-    gchar * red_color_str = g_strdup_printf("r c red");
-    gchar * black_color_str = g_strdup_printf("b c black");
+    gchar * green_color_str = g_strdup_printf("g c #2ECC40");
+    gchar * yellow_color_str = g_strdup_printf("y c #FFDC00");
+    gchar * red_color_str = g_strdup_printf("r c #FF4136");
+    gchar * black_color_str = g_strdup_printf("b c #111111");
```

Comment:
I would prefer to use named constants—see https://www.w3schools.com/colors/colors_hex.asp— or something defined in our css files.